lib/src/fields/form_builder_phone_field.dart

```diff
@@ -216,7 +216,7 @@ class FormBuilderPhoneField extends FormBuilderField {
   _FormBuilderPhoneFieldState createState() => _FormBuilderPhoneFieldState();
 }
 
-class _FormBuilderPhoneFieldState extends FormBuilderFieldState {
+class _FormBuilderPhoneFieldState extends FormBuilderFieldState<String> {
```

Comment:
`FormBuilderFieldState` should be changed to use a type for the widget.  I did this in the master -- you don't need to override `widget` getter for each `State` class.

Reply:
Oh perfect, I'll see that!

Reply:
(I realize that this idea is a separate commit, but thought I'd mention it because it popped out at me.)

Reply:
Ah okay, no problem.